### ISSUE TITLE
data-placeholder value is not identified by JAWS and NVDA

### ISSUE THREAD

**Issue by ananthula** [quilljs/quill#3725]
Please describe the a concise description and fill out the details below. It will help others efficiently understand your request and get to an answer instead of repeated back and forth. Providing a [minimal, complete and verifiable example](https://stackoverflow.com/help/mcve) will further increase your chances that someone can help.

**Steps for Reproduction**

1. visit quill playground(https://quilljs.com/playground/) and open JAWS/NVDA
2. focus on quill editor and observe that JAWS/NVDA does not read the placeholder text.

**Expected behavior**:
NVDA/JAWS should read the placeholder text

**Actual behavior**:
JAWS/NVDA does not read the placeholder text.
**Platforms**:

Include browser, operating system and respective versions

**Version**:
1.3.7
Run `Quill.version` to find out

**Comment by bBaxta** [comment]
Any plans to address this issue in the near future?
